### ISSUE TITLE
Warning: No remote package servers have been specified.

### ISSUE THREAD

**Issue by klausweiss** [typelead/etlas#85]
Hello,
I've just tried setting up an eta project using gradle (https://github.com/typelead/eta-init), but it hangs in the `:setupEnvironmentEta` step.

When run manually, `etlas` warns about not having found any remote package servers:
```
$ ~/.gradle/caches/etlas/1.5.0.0/etlas update
Warning: No remote package servers have been specified. Usually you would have
one specified in the config file.
```

What do I put in `~/.etlas/config` to define a remote server?

**Comment by rahulmutt** [comment]
Can you share your platform/OS details?

**Comment by klausweiss** [comment]
Sure. I'm on Xubuntu 18.04. Kernel 4.15.0-43-generic, but this is probably irrelevant.

**Comment by klausweiss** [comment]
I've resolved the issue. 

The problem was I ran `./gradlew` without the `-PetaSendMetrics` flag first, then read in the documentation how to enable/disable sharing metrics using the config file. 
`etlas` would then recognize my config file as present and valid although it only contained the `send-metrics` flag.

After temporarily deleting the `~/.etlas/config` file it has been automatically generated. It contains the default configuration including the two repositories (_hackage_ and _etlas-index_).
